examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -279,13 +279,13 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW
+ * @def OPENTHREAD_CONFIG_CSL_CRYSTAL_PPM
```

Comment:
This macro is not needed as the value could be got from radio API `otPlatTimeGetXtalAccuracy`